cfme/utils/net.py

```diff
@@ -180,6 +180,27 @@ def find_pingable(mgmt_vm, allow_ipv6=True):
         return getattr(mgmt_vm, 'ip', None)
 
 
+def find_pingable_ipv6(mgmt_vm):
+    """Looks for a pingable ipv6 address from mgmt_vm.all_ips
+
+     Assuming mgmt_vm is a wrapanapi VM entity, with all_ips and ip methods
+
+     Returns:
+         In priority: first pingable ipv6 address, address 'selected' by wrapanapi (possibly None)
+     """
+    for ip in getattr(mgmt_vm, 'all_ips', []):
+        if not is_pingable(ip) or not is_ipv6(ip):
```

Comment:
Given the context of the method, this logic should be flipped. Only check pingable state on IPv6 addresses, as you don't care if its pingable and IPv4, and will have wasted several seconds pinging the address.